pkgs/development/node-packages/node-packages.json

```diff
@@ -237,6 +237,7 @@
 , "redoc-cli"
 , "remod-cli"
 , "reveal.js"
+, "reveal-md"
```

Comment:
you will probably want to add `inherit (nodePackages) reveal-md;` to `pkgs/top-level/all-packages.nix` for this to be easy-to-find

Reply:
Thank you! Fixed.